markdown/chapters/meandeck/interaction.md

```diff
@@ -487,13 +515,13 @@ the turn building the following:
 
 This pile only actually casts two spells on the turn after resolving Doomsday.
 Ideas Unbound and the Thassa's Oracle itself. You can also utilise piles of just
-3 cycling effects or add in Cavern to the mix to protect against other effects
-should it be desired.
+three cycling effects or add in Cavern to the mix to protect against other
+effects should it be desired.
 
-### Thought Scour / Predict
+### Thought Scour / Predict / Archmage's Charm
 
-Thought Scour is not normally a common card in Legacy now that the Underworld
-Breach era is gone however it still can show up in rogue brews from time to
+Thought Scour is not normally a common card in Legacy now that the {{Underworld
+Breach}} era is gone however it still can show up in rogue brews from time to
```

Comment:
```suggestion
Breach}} era is gone however it can still show up in rogue brews from time to
```
Grammar: is that a typo or a correct wording? (same as above, feel free to dismiss these kinds of comment at will)